vm/src/types/structseq.rs

```diff
@@ -86,5 +86,16 @@ pub trait PyStructSequence: StaticType + PyClassImpl + Sized + 'static {
                 .into(),
             );
         }
+
+        class.set_attr(
+            ctx.intern_str("__match_args__"),
```

Comment:
Because `__match_args__` is a core language attribute, adding it to `vm/src/vm/context.rs` would be a good idea.
Then we can refer interned keyword directly.
```suggestion
            identifier!(vm, __match_args__),
```